### PR TITLE
spatial transformer layer

### PR DESCRIPTION
**Pull request by fabiansinz**

### CODE REVIEW

File: attorch/layers.py [hunk 1]
```diff
@@ -32,9 +29,11 @@ class Elu1(nn.Module):
     """
 
     def forward(self, x):
-        return elu1(x)
+        return F.elu(x, inplace=True) + 1.
```
Comment (eywalker) [inline attorch/layers.py]: why not just use `elu1(x)` defined below?

File: attorch/layers.py [hunk 2]
```diff
@@ -244,23 +245,14 @@ def initialize(self, init_noise=1e-3):
         if self.bias is not None:
             self.bias.data.fill_(0)
 
-    def sigma_l1(self):
+    def sigma_l1(self, offset=0.1):
```
Comment (eywalker) [inline attorch/layers.py]: what's the `offset` used for?

File: attorch/layers.py [hunk 3]
```diff
@@ -244,23 +245,14 @@ def initialize(self, init_noise=1e-3):
...
-    def sigma_l2(self):
-        return (self.sigma - self.sigma_eps).pow(2).mean()
 
     def feature_l1(self, average=True):
         if average:
             return self.features.abs().mean()
         else:
-            return self.features.abs().sum()
-             
-    def weight_l1(self, average=True):
```
Comment (eywalker) [inline attorch/layers.py]: why are you removing these? I believe I have code using these regularizations.

Comment (fabiansinz) [reply attorch/layers.py]: must have been a mistake

**Pull request merged**